src/test/java/org/commcare/util/GetJsonPropertyTest.java

```diff
@@ -0,0 +1,18 @@
+package org.commcare.util;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+import static org.junit.Assert.assertTrue;
+
+public class GetJsonPropertyTest {
+
+    @Test
+    public void getJsonProperty() {
+        String testObj1 = "{name: Sam}";
```

Comment:
I'm surprised this allows the object key to not be quoted. Are there any quote-related options in JSONObject? I'm guessing the data coming from web apps will have the keys quoted.

Reply:
I can check on that. I did wonder about it. Here I was going by example is the code.

Reply:
The docs I've found are rather vague. "source - A string beginning with { (left brace) and ending with } (right brace)." I tested it both ways and both work.